app/src/main/java/com/ndt/themoviedb/data/model/Movie.kt

```diff
@@ -0,0 +1,14 @@
+package com.ndt.themoviedb.data.model
+
+data class Movie(
+    var movieID: Int?,
```

Comment:
mấy biến này a gán 1 giá trị mặc định nào đó chứ nếu không thì sau mình lại phải sài assert not null (!!) a ơi. 

Mà theo convention của Kotlin ý cũng không cần đặt ngữ cảnh movie tại đầu mỗi biến tại người ta nhìn vào id là hiểu ID thuộc lớp Movie này r a